docs/releasing.md

```diff
@@ -0,0 +1,59 @@
+## Releasing a new version of the koku-metrics-operator
+
+More specific documentation for generating bundles for a release can be found [here](https://operator-framework.github.io/community-operators/testing-operators/).
```

Comment:
```suggestion
Before releasing a new version of the operator, testing should be performed as described [here](release-testing.md).
```
Let's reference our testing doc instead of the operator-framework docs. If someone comes to this doc first, they might overlook our more specific testing documentation.